packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -0,0 +1,96 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:ui';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/services.dart';
+
+import 'box.dart';
+import 'layer.dart';
+import 'object.dart';
+
+
+enum _PlatformViewState {
+  uninitialized,
+  resizing,
+  ready,
+}
+
+/// A render object for an Android view.
+///
+/// [RenderAndroidView] is responsible for sizing and displaying the Android view.
```

Comment:
the Android view => an Android View

Maybe View should be a link to https://developer.android.com/reference/android/view/View

Reply:
done